tests/packagedcode/test_package_models.py

```diff
@@ -22,23 +22,146 @@
 #  ScanCode is a free software code scanning tool from nexB Inc. and others.
 #  Visit https://github.com/nexB/scancode-toolkit/ for support and download.
 
-from __future__ import absolute_import, print_function
+from __future__ import absolute_import
+from __future__ import print_function
 
 import os.path
 
 from commoncode.testcase import FileBasedTesting
 
+from packagedcode.models import AssertedLicense
 from packagedcode import models
-from collections import OrderedDict
+from packagedcode.models import Package
+from packagedcode.models import Party
 
 
 class TestModels(FileBasedTesting):
     test_data_dir = os.path.join(os.path.dirname(__file__), 'data')
 
     def test_model_creation_and_dump(self):
-        aap = models.AndroidAppPackage()
+        aap = models.AndroidAppPackage(name='someAndroidPAcakge')
         result = aap.as_dict()
-        expected = OrderedDict([('type', 'Android app'),
-                                ('packaging', 'archive'),
-                                ('primary_language', 'Java')])
+        expected = {
```

Comment:
We need the as_dict() function to return an OrderedDict with a stable an well defined order. Otherwise the results of a identical scan may not be the same text-wise. We want them to be stable and in a predictable order.
Could you return an OrderedDict again?